### ISSUE TITLE
support for multiple zabbix servers

### ISSUE THREAD

**Issue by jjoe64** [inovex/zax#6]

**Comment by jimsmith** [comment]
Is the feature ready now as not sure why it is closed?

**Comment by Pimmetje** [comment]
Current i would say that multi server is so far broken that it is not useful (at least not for me)

**Comment by chrjsorg** [comment]
Please define broken.

**Comment by jimsmith** [comment]
Yeah very true in my case it just ends up not responding and crashes the app.

It should be noted to have it on road map ☺ as I have multiple different zabbix setup for different outfits and environments.

**Comment by Pimmetje** [comment]
If i switch servers it won't load any graph. The triggers have names from other servers. This all makes kind a mess. As i have always some triggers on both of the server i am using. (some users are lazy fixing there issues).
Please not that most of the issues start happening when u use widgets.

**Comment by jschamburger** [comment]
I wasn't aware of these issues. I just reopened the ticket as the feature is apparently not usable yet.

**Comment by jacentemu** [comment]
I have the same problem as @Pimmetje: when multiple zabbix servers are configured, the application  takes hostnames from the first configured or last modyfied zabbix server. Problem occurs both in application and in widget.When one zabbix servers has more hosts than other, application sometimes show "null" as hostname, because this host ID doesn't exists on one zabbix server.
Currently, when multiple servers are used, application is usable only because most warning messages contain also hostnames (eg. "Some {proces} is not working on {HOSTNAME}").

Other problem: when widget (configured to one zabbix server) is autoupdated sometimes it has warnings (and hostnames) from last modyfied or las used server (not configured in this widget).

**Comment by Pimmetje** [comment]
I even seen the problem about the null when i only have one server configured. So there is a reference failing somewhere. Ill still did not had the time to look at it myself.

**Comment by insanity54** [comment]
I've started a bounty for this feature. [![Bountysource](https://api.bountysource.com/badge/issue?issue_id=1776256)](https://www.bountysource.com/issues/1776256-support-for-multiple-zabbix-servers?utm_source=1776256&utm_medium=shield&utm_campaign=ISSUE_BADGE)

**Comment by alainseys** [comment]
Install zabbix proxy and you are good to go let the home network be monitored to the proxy and the proxy reports to the master
